nml/actions/action0.py

```diff
@@ -448,6 +448,8 @@ def get_property_info_list(feature, name):
     else: assert False
 
     for prop_info in prop_info_list:
+        if 'replaced_by' in prop_info:
+            generic.print_warning("Please consider replacing '{}' with '{}'".format(name.value, prop_info['replaced_by']), name.pos)
```

Comment:
This doesn't work properly when 'name' is a number. I have a patch for that.

Reply:
I applied your patch. And reworded other warnings too.